sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -92,13 +96,121 @@ class OrchestrationController {
             .inputParams(inputParams);
       };
 
+  private static final List<DPIEntityConfig> ALL_DPI_ENTITIES =
```

Comment:
(Major/Question)

Is there a reason not to iterate over `DPIEntities.values()` instead of maintaining the list twice?

Reply:
Fixed